src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmissingwhitespaceafterasterisk/InputJavadocMissingWhitespaceAfterAsterisk.java

```diff
@@ -0,0 +1,59 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmissingwhitespaceafterasterisk;
+class InputJavadocMissingWhitespaceAfterAsterisk
```

Comment:
please add 3 lines of empty lines before class we will reuse this in future when Check become to have properties and we need to put in that lines config.

Please do same for other inputs.